### ISSUE TITLE
leaking color picker instances

### ISSUE THREAD

**Issue by humanchimp** [RSSchermer/ember-spectrum-color-picker#10]
Hello,

ISTM that this addon leaks color picker instances. I am using 8 pickers, and I have a checkbox that toggles them off and on. Every time I uncheck and check the checkbox, it leaks 8 instances of the color picker that never get cleaned up. This appears to be because the color picker [is being added to the DOM on `didInsertElement`](https://github.com/RSSchermer/ember-spectrum-color-picker/blob/master/addon/components/spectrum-color-picker.js#L62), but no symmetrical teardown hook on `willDestroyElement` exists.

I'll try to send through a PR to fix this.

**Comment by RSSchermer** [comment]
Thanks for uncovering this. :+1: 

I'll look forward to the PR. If you hit a snag I'd also be happy to look into this myself, just let me know.

**Comment by riklaunim** [comment]
```
willDestroyElement: function() {
    this.$().spectrum("destroy");
  }
```

Seems too prevent leaving a invisible div at the end of the DOM.

**Comment by RSSchermer** [comment]
Thanks @riklaunim, this issue dropped of my radar. I just published 0.4.2 which should include your fix.
